docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -334,3 +334,22 @@ should be executed before this changes.
 - `spec.infrastructureTemplate` has been moved to `spec.machineTemplate.infrastructureRef`. Thus, cluster templates which include `KubeadmControlPlane`
 have to be adjusted accordingly.
 - `spec.nodeDrainTimeout` has been moved to `spec.machineTemplate.nodeDrainTimeout`.
+
+## Required spec and status fields for Control Planes using 'version' for ClusterClass support
+ControlPlane implementations using version must now [include a 'version' field as defined below in both its spec and its status](../architecture/controllers/control-plane.md#required-spec-fields-for-implementations-using-version).
+
+ `version` - is a string representing the Kubernetes version to be used
```

Comment:
Maybe it's better to drop the duplicate definition here as `.status.version` is defined differently

Reply:
Thanks for pointing that out! I've updated with a separate definition for each field which should be useful to someone consuming these docs I think.